package/MDAnalysis/topology/guessers.py

```diff
@@ -225,7 +225,7 @@ def guess_atom_element(atomname):
         return no_symbols
 
 
-def guess_bonds(atoms, coords, box=None, **kwargs):
+def guess_bonds( atoms, coords, box=None, **kwargs):
```

Comment:
```suggestion
def guess_bonds(atoms, coords, box=None, **kwargs):
```
Why is this empty space being added?